### PR TITLE
Add script for getting changelog entries from the merge commits

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Signed-off-by: Frantisek Lachman <flachman@redhat.com>

### CODE REVIEW

File: scripts/changelog.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    The script goes through the merge commits since the specified REF
+    and get the changelog entry from the commit message.
+    By now, we parse a last paragraph of the pull-request description
+    (that is contained in the commit message).
+    In the future, we will have an explicit divider.
+    """,
+)
+@click.argument("ref", type=click.STRING)
+def changelog(ref):
+    print(get_changelog(get_relevant_commits(Repo("."), ref)))
```
Comment (lbarcziova) [inline scripts/changelog.py:46]: what about being able to specify the path to the repo?

File: scripts/changelog.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+def get_relevant_commits(repository: Repo, ref: str) -> List[Commit]:
+    range = f"{ref}..HEAD"
+    return list(repository.iter_commits(rev=range, merges=True))
+
+
+def convert_message(message: str) -> str:
+    cleared_message = message.split("Reviewed-by")[0].strip()
+    return cleared_message.split("\n\n")[-1].strip()
```
Comment (lbarcziova) [inline scripts/changelog.py:21]: nice simplification!

**Review by lbarcziova — APPROVED** [review]
LGTM, just one suggestion

**Review by FrNecas — COMMENTED** [review]
Overall looks good to me, just one suggestion

File: scripts/changelog.py (hunk 1 above)
Comment (lachmanfrantisek) [reply scripts/changelog.py]: OK, fixed.

File: scripts/changelog.py (hunk 2 above)
Comment (lachmanfrantisek) [reply scripts/changelog.py:21]: It also fixed the issue for commits without `Reviewed-by`.

**Review by TomasTomecek — APPROVED** [review]

**Comment by lachmanfrantisek** [comment]
I am going to merge this. Please, create an issue or send PR (..;) for other improvements. (And I am sure this will need some care...;)

**Pull request merged**